_projects/2024/100535255/100535255.Rmd

```diff
@@ -0,0 +1,1059 @@
+---
+title: "Degrees and Nobels: A non-essential link"
+description: |
+  A comprehensive look at Nobel Prizes from 1901 to 2012, uncovering patterns 
+  in laureates ages, trends across categories, educational qualifications, academic 
+  affiliations and hometown demographics.
+categories: "2024"
+author: "Andrea Rivera-Mateos"
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="200%", fig.align="center")
+```
+
+```{css text alignment}
+#| echo: false
+p {text-align: justify}
+```
+
+## **Setting the stage**
+
+The Nobel Prizes, awarded annually since 1901, honor extraordinary achievements in various fields, including Physics, Chemistry, Medicine, Literature, Peace, and Economic Sciences. These prestigious awards not only celebrate groundbreaking contributions but also reveal fascinating patterns and stories about the individuals who have shaped human progress.
+
+This visualization delves into the journey of Nobel laureates over time, capturing the diversity of achievements and backgrounds that define these awards. It highlights trends in age at the time of recognition, the global origins and affiliations of the winners, and how different categories have evolved through the years. By examining these elements, the visualization provides a deeper understanding of the context in which these honors were awarded, the historical moments they reflect, and the enduring impact of the laureates' work.
+
+The decision to explore this graph was inspired by the intricate narratives it weaves, capturing the interplay of time, remarkable achievements, and the diverse tapestry of humanity. Through its depiction of Nobel laureates, the visualization illuminates the historical and personal contexts behind these prestigious awards, offering a window into the evolving legacy of innovation and excellence. It invites reflection on the profound stories hidden within the data, highlighting how visual exploration can bring to life the enduring impact of these extraordinary individuals.
+
+For this visualization, Georgia Lupi, author of the original graph, took her inspiration from musical scores and their elegant aesthetics, especially John Cage's source of fascinating contemporary scores. She visualized the evolution over time of the 6 prize categories, as colored musical scores along a skewed timeline.
+
+![*Nobels, no degrees* by Georgia Lupi for [La Lettura](https://www.flickr.com/photos/accurat/8249872482/in/album-72157632180303367/), Corriere della Sera. (Nov 25, 2012)](nobels.jpg){width="100%"}
```

Comment:
Please set it as external as e.g. David does [here](https://github.com/IBiDat/dataviz/blob/9a3bc0367b1c0b297765b59ff6f3c34cef06a204/_projects/2024/100535712/100535712.Rmd#L26).